src/uagents/models.py

```diff
@@ -1,20 +1,61 @@
 import hashlib
-from typing import Type, Union
+from typing import Type, Union, Dict, ClassVar, Any
 
 from pydantic import BaseModel
+from pydantic.schema import model_schema, default_ref_template
 
 
 class Model(BaseModel):
+    schema_no_descriptions: ClassVar[Union[Dict[str, Any], None]] = None
+
+    @staticmethod
+    def _remove_descriptions(
+        model: Type["Model"], orig_descriptions: Dict[str, Union[str, Dict]]
+    ):
+        for field_name, field in model.__fields__.items():
+            if field.field_info and field.field_info.description:
+                orig_descriptions[field_name] = field.field_info.description
+                field.field_info.description = None
+            elif issubclass(field.type_, Model):
+                orig_descriptions[field_name] = {}
+                Model._remove_descriptions(field.type_, orig_descriptions[field_name])
+
+    @staticmethod
```

Comment:
From my point of view these methods shouldn't be static  most of the times the model is even passed in why not use `self` then?

Reply:
The model param passed to these newly created methods is actually not an instance of a Model subclass but rather a Model subclass type (so a class object) that's why I couldn't implement these methods as instance methods (so using self.). However you are right in the sense that these method could be class methods using cls.

Reply:
In the refactored version I created class methods instead of static methods where possible, please check it.